lib/travis/shell/generator/bash.rb

```diff
@@ -144,7 +144,7 @@ def handle_else(cmds)
         private
 
           def handle_secure_vars(key, value, options)
-            if options[:echo] && options.delete(:secure)
+            if options[:echo] && options[:secure]
```

Comment:
The `delete` is moved here so you can pass along the `--secure` in https://github.com/travis-ci/travis-build/pull/942/files#diff-8c5fd04469601ade1797d06043753110R37 

Do we still need to delete it at another point? Or are we safe to leave the `.delete` off?